### ISSUE TITLE
When saving tiff, .tif occurs twice in filename

### ISSUE THREAD

**Issue by marthinwurer** [ozak/georasters#46]
In code:
```
    one_deg.to_tiff(os.path.expanduser("~/path/to/dir/one_deg.tif"))
```
In file:
```
$ ls
one_deg.tif.tif
```

I expected that the file would just be named `one_deg.tif`, not `.tif.tif`

**Comment by ozak** [comment]
Well since it is a ``to_tiff`` method, it assumes the you want the ending to be ``.tif``. You do not need to assign it. I guess it is a preference issue.

**Comment by marthinwurer** [comment]
I could throw together a PR to check the ending of the path given, and just not add it if it's like `tif` or `tiff` or capitalized versions of those. Would that be acceptable?

**Comment by ozak** [comment]
sure!

**Comment by ozak** [comment]
I guess one could just put a ``replace`` on the string to delete any ``tif/tiff/TIF/TIFF`` endings

**Comment by marthinwurer** [comment]
Yeah, there are a few things that could be done. I'll take a look at the code and see which will work best.
